src/dotnet/APIView/APIViewWeb/Managers/NotificationManager.cs

```diff
@@ -16,6 +16,7 @@
 using Microsoft.ApplicationInsights;
 using System.Net.Http;
 using System.Net.Http.Json;
+using Newtonsoft.Json;
```

Comment:
Why newtonsoft over System.Text.Json? Or is newtonsoft already pretty heavily used in this project?

Reply:
It's already used. I will change it to use System.Text.Json. Thanks

Reply:
Done.

Reply:
System.Text.Json has some issues when used with cosmos db.